content/docs/contributing/contributing_changes.md

```diff
@@ -29,133 +27,148 @@ In short
 
 This workflow should help changes be deployed quickly.
 
+## Creating a Branch
+
+You should make your changes on a new branch, and not on `main`.
+This will make it easier to merge changes from many contributors.
+Try to pick a branch name which is short and describes the change you are making.
+
+{{% hint warning %}}
+The changes on one branch or pull request should address a
+single issue and be self-contained.
+Don't try to solve more than one unrelated problem at once.
+
+Sticking to this practice will help ensure pull requests are small and easy to review.
+Otherwise, you might end up having to make many changes during review, have difficultly keeping your branch up to date or upset the reviewers.
+{{% /hint %}}
+
+## Making Changes
+
+At this point, you can edit files and create `git` commits as necessary.
+
 {{% hint info %}}
 When making commits, it is a good idea to follow Tim Pope's
 [recommendations](https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html)
 for good commit messages.
 {{% /hint %}}
 
-## Creating a Branch
+It is often useful to run the Markdown style checker (and other tests) before committing.
+This allows you to catch, and fix, potential issues before opening a pull request.
+To do so, use the `pre-commit` tool.
+From the repository root, run:
+
+```bash
+brew install pre-commit  # if not already installed
+pre-commit install
+```
 
-You should make your changes on a new branch, and not on `main`. This will make
-it easier to merge changes from many contributors. Try to pick a branch name
-which is short and describes the change you are making.
+This creates a pre-commit [Git hook](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks),
+which (as the name suggests) will be automatically run every time before a commit is created.
+It currently performs the following checks, which are designed to mimic those in [Continuous Integration](#continuous-integration):
 
-{{% hint warning %}} The changes on one branch or pull request should address a
-single issue and be self-contained. Don't try to solve more than one unrelated
-problem at once.
+- Markdown linting, i.e. style checking:
+- a Hugo build, to make sure that the pages can be built; and
+- a Markdown link check.
 
-Sticking to this practice will help ensure pull requests are small and easy to
-review. Otherwise, you might end up having to make many changes during review,
-have difficultly keeping your branch up to date or upset the reviewers.  {{%
-/hint %}}
+If you want to manually perform these checks at any point in time, use:
```

Comment:
I think it might be worth mentioning that -- if I understand correctly -- this will run the checks over files that have changes committed or staged (rather than running it over everything).